src/_common/scroll/scroller/scroller.ts

```diff
@@ -10,65 +9,24 @@ import { AppScrollInviewParent } from '../inview/parent';
 	},
 })
 export default class AppScrollScroller extends Vue {
-	@Prop(Boolean)
-	overlay?: boolean;
+	@Prop(propOptional(Boolean))
+	thin?: boolean;
 
-	@Prop(Boolean)
+	@Prop(propOptional(Boolean))
 	horizontal?: boolean;
 
-	@Prop(Boolean)
+	@Prop(propOptional(Boolean))
 	hideScrollbar?: boolean;
 
 	isMounted = GJ_IS_SSR;
-
-	// Underscored so that vue doesn't watch.
-	private _scrollElement!: HTMLElement;
-	private _isDestroyed?: boolean;
-	private _simplebar?: SimpleBar;
-
-	get shouldOverlay() {
-		return this.overlay && !GJ_IS_SSR;
-	}
-
-	/**
-	 * The actual Element that scrolls for this Scroller.
-	 */
-	get scrollElement() {
-		return this._scrollElement;
-	}
+	_scrollElement!: HTMLElement;
```

Comment:
You can actually get rid of this too, just keep the isMounted. Then in the template you can just pass in $el to the child prop. We just need the ismount to make sure we render it when we're mounted.